### PR TITLE
Docs/readme update

### PR DESCRIPTION
**Pull request by TheFutonEng**
## Description

This PR gets a `Makefile` in the repo and the start of an `ansible` folder in order to make it easy to execute automation in multiple environments. 

## Related Issue

Fixes #

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,74 @@
...
+uds bundle create
+```
 
 ## Deploy
 
-`uds bundle deploy`
+The `create` step writes a UDS bundle to disk that needs to be referenced in the `deploy` step:
+
+```bash
+uds deploy uds-bundle-raptor-uds-amd64-0.0.1.tar.zst
```
Comment (andrewg-xyz) [inline README.md:67]: suggest x.y.z for versions. This precludes us from making a change to readme on every release OR if we forget someone deploying an old version inadvertently.

**Review by andrewg-xyz — COMMENTED** [review]
Does it make sense to move `ansible` into a sub folder `infra/ansible`?

My (nitpick) preference is to have top level folders be organizational. i.e. `src` `infra` `docs`

**Comment by TheFutonEng** [comment]
This works for me, will create an `infra` folder.

> Does it make sense to move `ansible` into a sub folder `infra/ansible`?
> 
> My (nitpick) preference is to have top level folders be organizational. i.e. `src` `infra` `docs`

**Comment by TheFutonEng** [comment]
Also noticed that the readme needs a lot of updates now.  Will push those and then ping you @andrewg-xyz .

**Comment by TheFutonEng** [comment]
@andrewg-xyz , give it another peek please.

**Review by andrewg-xyz — APPROVED** [review]

**Pull request merged**